src/Exceptions/HttpUnavailableForLegalReasons.php

```diff
@@ -8,4 +8,3 @@ class HttpUnavailableForLegalReasons extends HttpException {
     public const DESCRIPTION = "This status code indicates that the server is denying access to the resource in response to a legal demand.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/451";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces